cockpit_api/src/test/java/com/cockpit/api/service/jiragateway/JiraGatewayServiceTest.java

```diff
@@ -56,17 +65,64 @@ public void whenUpdateTotalNbUserStoriesTaskRuns_thenUpdateTotalNbUserStoriesInS
         mockJiraList.add(mockJira);
         List<Sprint> mockSprintList = new ArrayList<>();
         mockSprintList.add(mockSprint);
-
         Mockito.when(jiraRepository.findAllByOrderById()).thenReturn(mockJiraList);
         Mockito.when(sprintRepository.findByJiraOrderBySprintNumber(Mockito.any(Jira.class))).thenReturn(mockSprintList);
         Mockito.when(userStoryRepository.countUserStoriesByJiraAndCreationDateBefore(Mockito.any(Jira.class), Mockito.any(Date.class))).thenReturn(2);
         Mockito.when(userStoryRepository.countUserStoriesByJiraAndCreationDateGreaterThanAndCreationDateLessThanEqual(Mockito.any(Jira.class), Mockito.any(Date.class),  Mockito.any(Date.class))).thenReturn(3);
 
-        // when
+//      when
         jiraGatewayService.setTotalNbOfUserStoryForEachSprintOfEachProject();
 
         // then
         Assert.assertEquals(5, mockSprint.getTotalNbUs().intValue());
     }
 
+    @Test
+    public void whenDeleteOnJira_thenDeleteOnCockpit(){
+        Jira mockJira = new Jira();
+        Sprint mockSprint = new Sprint();
+
+        mockJira.setId(99999L);
+        mockJira.setJiraProjectKey("TEST");
+
+        mockSprint.setJira(mockJira);
+        mockSprint.setId(99999L);
+        mockSprint.setJiraSprintId(10024);
+        mockSprint.setSprintNumber(2);
+
+        UserStory mockUserStory = new UserStory();
+
+        mockUserStory.setJira(mockJira);
+        mockUserStory.setId(99999L);
+        mockUserStory.setSprint(mockSprint);
+
+        jiraRepository.save(mockJira);
```

Comment:
You are testing repositories here, have you considered to mock it with Mockito.when().thenReturn();